tools/mason/MasonModules.chpl

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+use ArgumentParser;
+use FileSystem;
+use List;
+use Map;
+use MasonUtils;
+use MasonHelp;
+use MasonEnv;
+use MasonBuild;
+use Subprocess;
+use TOML;
+
+proc masonModules(args: [] string) throws {
```

Comment:
could you add some comment here about the expected output of a `mason modules` call?